fabric/src/main/resources/data/ad_astra/patchouli_books/astrodux/en_us/entries/the_first_launch/your_first_rocket.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Your First Rocket",
   "icon": "ad_astra:nasa_workbench",
-  "sortnum": 0,
```

Comment:
This sortnum was duplicated with `basic_materials`'s. Although it has no bug IN-GAME, but I think it's better to fix it.